craft_cli/messages.py

```diff
@@ -44,15 +45,25 @@ def get_terminal_width():
     return shutil.get_terminal_size().columns
 
 
+def get_log_filepath(appname):
+    """Provide a filepath for logging into."""
+    # XXX Facundo 2021-09-03: this will change heavily in next couple of branches
+    _, filepath = tempfile.mkstemp(prefix=f"charmcraft-{appname}-log-")
```

Comment:
isn't appname supposed replace charmcraft here?

Reply:
can we document appname?

Reply:
**this will change heavily in next couple of branches**

Added a line about "appname" in the big PR.

Reply:
Removed "charmcraft" from here, anyway.